static/js/search_suggestion.js

```diff
@@ -121,7 +121,7 @@ function get_stream_suggestions(last, operators) {
         const prefix = "stream";
         const highlighted_stream = highlight_query(regex, stream);
         const verb = last.negated ? "exclude " : "";
-        const description_html = verb + prefix + "&nbsp;" + highlighted_stream;
+        const description_html = verb + prefix + " " + highlighted_stream;
```

Comment:
Reverting this change made in #22307 as this partially fixed (more like hid) the bug that this commit fixes completely